app/eq.py

```diff
@@ -113,7 +115,7 @@ async def build(self):
             'language_code': self._language_code,
             'display_address':
             self.build_display_address(self._sample_attributes),
-            'response_id': self._response_id,
```

Comment:
The definition of self._response_id needs removing if it is not being used any more

Reply:
yes, remove the code setting response_id to questionnaire_id at about line 66